### ISSUE TITLE
Movie Recommender System : Generate Recommendations using Cosine Similarity

### ISSUE THREAD

**Issue by TamoghnoBhattacharya** [GeekHaven/HacktoberFest-AI#30]
We will implement our first content-based model now. Our model will consist of a function that accepts the movie name and number of recommendations to generate. All code should be within the function (except importing libraries). Follow the given procedure -
1. Use the code created in issue #2, available under Movie Recommender System/IIT2019139/metadata_prep.py to generate the metadata dataframe.
2. Use TF-IDF vectorizer to generate embedding vectors for each movie on basis of metadata.
3. Use cosine similarity to generate the similarity values between given movie and all other movies.
4. Pick out the movies with the top scores, except itself (which is highest of course, i.e. 1). Number of movies to pick is given by 'number of recommendations' argument. 
5. Return a dataframe consisting of 2 columns - the recommended movies and their similarity scores.

**Comment by JLodha** [comment]
I want to work on this issue

**Comment by TamoghnoBhattacharya** [comment]
@JLodha Go ahead.

**Comment by divyanshrai655** [comment]
@JLodha are you still working on this? You can seek help over discord if you want

**Comment by JLodha** [comment]
> @JLodha are you still working on this? You can seek help over discord if you want

Yes I am currently working on it. The work was delayed due to PAS Quiz and ITM Presentation.

Expecting a little time more. I hope nobody have any problem with this
